src/DIRAC/WorkloadManagementSystem/Service/PilotManagerHandler.py

```diff
@@ -75,9 +75,7 @@ def export_addPilotTQReference(
         cls, pilotRef, taskQueueID, ownerDN, ownerGroup, broker="Unknown", gridType="DIRAC", pilotStampDict={}
     ):
         """Add a new pilot job reference"""
-        return cls.pilotAgentsDB.addPilotTQReference(
-            pilotRef, taskQueueID, ownerDN, ownerGroup, broker, gridType, pilotStampDict
-        )
+        return cls.pilotAgentsDB.addPilotReference(pilotRef, ownerDN, ownerGroup, broker, gridType, pilotStampDict)
```

Comment:
The name of the method should then be changed to `export_addPilotReference`, and `taskQueueID` would not be needed anymore. But really it should be better to then deprecate the old method name (could be done in v8r0).